apps/machine/microblaze_generic/openamp-linux-u-MicroBlaze.dtsi

```diff
@@ -0,0 +1,36 @@
+/ {
```

Comment:
I 'm not sure that open-amp project is the right place for a DT file.  Adding this create a dependency with the Linux Kernel, and i suppose that it is a Xilinx Linux kernel distribution. 
Would be better to add this in Xilinx distribution.

Reply:
so I can remove this, but it is helpful for someone that wants to have this communicate with Linux and not have to derive the device tree. We can add a disclaimer in the README as well if you wish to see this denoted more explicitly as to be for just convenience and not a strict sign of formal support

Reply:
Yes agree that it  can be useful, but if integrated here, we will have to support compatibiliies with Linux Kernel and Yocto device tree overlay.
If you have no other place to put this file than OpenAMP repo, it would probably be best to merge this file in the README instead of directly creating a dtsi file.
You could put it just after mentioning the yocto devictree layer.

@edmooring : any opinion on that?